assets/stylesheets/common/locations.scss

```diff
@@ -612,3 +612,7 @@ input.input-location, div.input-location {
   height: calc(100vh - 270px);
   margin-top: 5px;
 }
+
+span.map-location{
+  width: 58%;
```

Comment:
Likewise, this is not a good solution. It breaks depending on the length of the address. Make sure, you try and think about all the different kinds of input your css needs to support. Setting specific widths like this is often not the right answer. Read up a bit more on flex boxes. They are often useful in situations like this: https://css-tricks.com/snippets/css/a-guide-to-flexbox/